src/components/SingleCard.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import '../styles/SingleCard.css';
+
+const SingleCard = ({
+  temp,
+  humidity,
+  condition,
+  icon,
+}) => (
+
+  <div className="Card">
+    <h4>{condition}</h4>
+    <img src={icon} alt="weather icon" />
+    <div className="Report">
+      <p>
+        {temp}
+        ℃
+      </p>
+      <p>
+        {humidity}
+        %
+      </p>
+    </div>
+  </div>
+);
```

Comment:
- [ ] [REQUIRED] You have put a great effort into this project. I can see that whenever search for a state of the country Nigeria, it updates the state and also fetches my current location which is very good. I also discovered that your details are been displayed on that same page. However, note that this project requires that you put the state of the country on the homepage and display its details on the details page. See again the image of the template provided for this project and as I have explained to you on our last call.
### Template Provided
![Screenshot from 2022-08-04 18-36-31](https://user-images.githubusercontent.com/58692590/182915134-1b210885-f95a-42a2-ad11-ff0aaefde314.png)